seqr/views/utils/terra_api_utils.py

```diff
@@ -192,10 +194,55 @@ def user_get_workspace_acl(user, workspace_namespace, workspace_name):
               "canCompute": true
             }
           }
+
     """
     path = "api/workspaces/{0}/{1}/acl".format(workspace_namespace, workspace_name)
     try:
         return _user_anvil_call('get', path, user).get('acl', {})
+    # Exceptions are handled to avoid error when a non-anvil user trying to validate permissions on AnVIL
     except (TerraNotFoundException, PermissionDenied) as et:
         logger.warning(str(et))
         return {}
+
+
+def update_workspace_acl(user, workspace_namespace, workspace_name, acl):
```

Comment:
I don't think adding this as a helper function is really helpful, as its never accessed outside this file and is only used one time and is only 2 lines. I think it would be more readable to just put the code in the `add_service_account` function itself